libraries/freertos_plus/standard/freertos_plus_tcp/include/FreeRTOSIPConfigDefaults.h

```diff
@@ -375,6 +383,14 @@ from the FreeRTOSIPConfig.h configuration header file. */
 	#endif /* _WINDOWS_ */
 #endif /* ipconfigMAXIMUM_DISCOVER_TX_PERIOD */
 
+#if( ipconfigUSE_DNS == 0 )
+	/* The DNS module will not be included. */
```

Comment:
The existing `#error` is there in order to prevent a compilation error, which was reported on [the forum](https://sourceforge.net/p/freertos/discussion/382005/thread/30508d072e/?limit=25#610a).
Enabling call-backs or the DNS cache doesn't lead to compilation errors or to confusion.